tidb-configuration-file.md

```diff
@@ -685,6 +695,70 @@ Configuration items related to read isolation.
+- Default value: `1`
+- Value options: `1` (enable, default) or `0` (disable).
+- The value of this configuration will initialize the value of system variable [`tidb_record_plan_in_slow_log`](/system-variables.md#tidb_record_plan_in_slow_log)
+- Before v6.1.0, this configuration is set by `record-plan-in-slow-log`.
+
+### `tidb_force_priority`
+
+- This configuration is used to change the default priority for statements executed on a TiDB server.
+- Default value: `NO_PRIORITY`
+- The default value `NO_PRIORITY` means that the priority for statements is not forced to change. Other options are `LOW_PRIORITY`, `DELAYED`, and `HIGH_PRIORITY` in ascending order.
+- Before v6.1.0, this configuration is set by `force-priority`.
+
+### `max_connections`
+
+- The maximum number of connections permitted for a single TiDB instance. It can be used for resources control.
+- Default value: `0`
+- Range: `[0, 100000]`
+- Defaultly the value of `0` means no limit. When the value of this variable is larger than `0`, and the number of connections reaches the value, TiDB server would reject new connections from clients.
+- The value of this configuration will initialize the value of system variable [`max_connections`](/system-variables.md#max_connections)
+- Before v6.2.0, this configuration is set by `max-server-connections`.
+
+### `tidb_memory_usage_alarm_ratio`
+
+- TiDB triggers an alarm when the memory usage of tidb-server instance exceeds a certain threshold. The valid value for this configuration item ranges from `0` to `1`. If it is configured as `0` or `1`, this alarm feature is disabled.
+- Default value: `0.8`
+- When the memory usage alarm is enabled, if [`server-memory-quota`](/tidb-configuration-file.md#server-memory-quota-new-in-v409) is not set, then the threshold of memory usage is ```the `memory-usage-alarm-ratio` value * the system memory size```; if `server-memory-quota` is set to a value greater than 0, then the threshold of memory usage is ```the `memory-usage-alarm-ratio` value * the `server-memory-quota` value```.
+- When TiDB detects that the memory usage of the tidb-server instance exceeds the threshold, it considers that there might be a risk of OOM. Therefore, it records ten SQL statements with the highest memory usage, ten SQL statements with the longest running time, and the heap profile among all SQL statements currently being executed to the directory [`tmp-storage-path/record`](/tidb-configuration-file.md#tmp-storage-path) and outputs a log containing the keyword `tidb-server has the risk of OOM`.
+- The value of this configuration will initialize the value of system variable [`tidb_memory_usage_alarm_ratio`](/system-variables.md#tidb_memory_usage_alarm_ratio)
+- Before v6.1.0, this configuration is set by `memory-usage-alarm-ratio`.
+
+### `tidb_enable_ddl`
+
+- This configuration controls whether the corresponding TiDB instance can run DDL statements or not.
+- Default value: `ON`
```

Comment:
It's true in the Chinese version.

Reply:
Then the Chinese version is incorrect....

Reply:
```suggestion
- Default value: `true`
```